### PR TITLE
Add country to digital wallet settings

### PR DESCRIPTION
**Pull request by colinbird**

### CODE REVIEW

File: openapi/components/schemas/Country.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+description: The ISO 3166 alpha-2 country code.
+maxLength: 2
```
Comment (slavcodev) [inline openapi/components/schemas/Country.yaml:2]: This constraint does not make sense along with specified `pattern`. But if you'd like to leave it, consider to add `maxLength: 2` then.

File: openapi/components/schemas/PaymentCards/DigitalWallets.yaml [hunk 2]
```diff
@@ -34,3 +36,7 @@ properties:
         description: The merchant origin in Google Pay. The fully qualified domain name.
         type: string
         example: "www.example.com"
+      country:
+        $ref: ../DigitalWalletCountry.yaml
+        description: TEST The ISO 3166 alpha-2 country code where the payment is processed. This is often the location for settling the payment. Consult with your payment service provider (PSP) to determine the appropriate country code.
```
Comment (slavcodev) [inline openapi/components/schemas/PaymentCards/DigitalWallets.yaml]:
```suggestion
        $ref: ../DigitalWalletCountry.yaml
```

**Review by ashkarpetin — APPROVED** [review]

**Review by dara123 — APPROVED** [review]

**Review by n-fasano — APPROVED** [review]

File: openapi/components/schemas/Country.yaml (hunk 1 above)
Comment (colinbird) [reply openapi/components/schemas/Country.yaml]: Thank you @slavcodev :)

File: openapi/components/schemas/PaymentTokens/DigitalWalletValidation/ApplePayValidation.yaml [hunk 3]
```diff
@@ -11,7 +11,7 @@ allOf:
         description: The validation request.
         type: object
         properties:
-          validationURL:
+          validationUrl:
```
Comment (rensftw) [inline openapi/components/schemas/PaymentTokens/DigitalWalletValidation/ApplePayValidation.yaml:14]: Weird, FramePay has been using `validationUrl` all this time 🤔

**Review by rensftw — APPROVED** [review]

**Review by adamaltman — APPROVED** [review]
Thank you.

**Pull request merged**